hw/sensor/creator/src/sensor_creator.c

```diff
@@ -299,17 +299,30 @@ static struct sensor_itf i2c_0_itf_ms40 = {
 };
 #endif
 
-
 #if MYNEWT_VAL(I2C_0) && MYNEWT_VAL(BMA253_OFB)
-static struct sensor_itf i2c_0_itf_lis = {

```

Comment:
Why is there a need to change this to SPI if the driver is working over I2C. We want to keep it as I2C.